app/connectors/AddressLookupConnector.scala

```diff
@@ -32,7 +33,9 @@ class AddressLookupConnectorImpl @Inject()(http: HttpClient, config: FrontendApp
     val addressLookupUrl = s"${config.addressLookUp}/v2/uk/addresses?postcode=$postCode"
 
     http.GET[Seq[AddressRecord]](addressLookupUrl)(implicitly, schemeHc, implicitly).map(Some(_)) recoverWith {
-      case _ => Future.successful(None)
+      case t: Throwable =>
+        Logger.error("cannot connect to address lookup service", t)
```

Comment:
It'd be a good idea to be a bit more explicit here, ie actually log the error response?  The error might not necessarily be that we can't connect to the address lookup service.